app/views/options/index.html.haml

```diff
@@ -41,6 +52,27 @@
         quick_donation_redirect |
         donation_ack_from |
         ) |
+      .form-row
+        .col-md-4.text-right
+          %label.col-form-label Allow Monthly Recurring Donations
+          = option_description_for(:allow_recurring_donations)
+        .col-md-8{id:"allow_recurring_donations_select"}
+          = f.select :allow_recurring_donations, [['Yes', true], ['No', false]], {}, {:class => 'form-control', onchange: 'showDefaultDonation(this)'}
+      .form-row{id:"default_donation_type_form_row", style:"visibility: #{@o.allow_recurring_donations ? 'visible' : 'hidden'}"}
+        .col-md-4.text-right
+          %label.col-form-label Default Donation Type
+          = option_description_for(:default_donation_type)
+        .col-md-8{id:"donation_type_radio"}
+          .radio-group.form-inline
+            = f.radio_button :default_donation_type, 'one', :class => 'form-control', :id => 'one'
+            = f.label :default_donation_type, 'One Time', :class => 'form-control', :for => 'one'
+            = f.radio_button :default_donation_type, 'monthly', :class => 'form-control', :id => 'monthly'
+            = f.label :default_donation_type, 'Monthly', :class => 'form-control', :for => 'monthly'
+      = render_collection_of_options f, %w( |
+        recurring_donation_contact_emails |
+        notify_theatre_about_new_recurring_donation |
```

Comment:
theater is misspelled twice in two differnt ways on these two different lines .  notice that it is spelled 'theater' throughout the app. when there are multiple ways to spell something, always prefer the one that already exists in the codebase.  also line 74 is just an error.